App.tsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  Platform,
+  StyleSheet
+} from 'react-native';
+import NfcManager, {NfcTech} from 'react-native-nfc-manager';
+import {Buffer} from 'buffer';
+
+interface State {
+  apduResponse: string;
+  testOutput: boolean;
+  detectedTechnology: string;
+  nfcUID: string;
+  showInstructions: boolean;
+}
+
+export class App extends React.Component<any, State> {
```

Comment:
To keep the codebase homogeneous you could use functional component instead of class components.
They are more concise and there is less boilerplate code

[Here an example](https://github.com/pagopa/io-app/blob/0832570a74b512602840635039b0a101d396c8aa/ts/features/bonus/bonusVacanze/components/TosBonusComponent.tsx#L60) (note useEffect does the same of `componentDidMount` + `componentWillUnmount`)

Reply:
You should [set your IDE](https://www.robinwieruch.de/how-to-use-prettier-vscode) to use prettier.
It ensures your code is well formatted

Reply:
Thank you for pointing that out, I will refactor it in a more "functional" style.
I had already installed Prettier, but it wasn't my default code formatter in VSC.. my bad